azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/functions/ParseJobStatus.java

```diff
@@ -30,6 +30,7 @@
       DONE,
       IN_PROGRESS,
       FAILED,
+      NO_CONTENT,
```

Comment:
Returns 204-No Content if resource does not exist. https://msdn.microsoft.com/en-us/library/azure/mt163632.aspx Used by `PublicIPAddress`, 'SubnetApi`, and `VirtualNetworkApi`.